report/src/introduction/waterfall_vs_agile.tex

```diff
@@ -1,2 +1,60 @@
 %This section should describe different approaches to handling software development complexity
-%It should highlight that both approaches are possible but one is more suitable.
\ No newline at end of file
+%It should highlight that both approaches are possible but one is more suitable.
+
+Developing a software product for a business can be a complex process.
+It can be useful to use software process models to guide the development process.
+In practice, there are two predominant models: the plan-driven waterfall model, and the incremental agile model.
+
+These frameworks can be extended and adapted to create more specific software engineering processes.
+
+The waterfall model has development cascade from one phase to the next, which is what gives the model its name.
+In broad terms, development starts with defining requirements, proceeds to a design phase, then to implementation, then testing, and lastly operation and maintenance.
+That is, development happens in stages. The next phase doesn't start until the previous phase is finished.
+
+However, this isn't often the case. Problems can occur during development, leading to delays. Each time new information emerges, the team has to go back to the previous stages and update design documents. The customer also needs to sign off on these, which contributes to the delays.
+It is also possible to simply freeze the softwware specification. Then, no changes can be made. This would, however, also mean that the identified problems aren't fixed. In the end, this could result in the system not behaving as the user desired. It can also lead to technical debt, as the development team has to somehow navigate around the identified problems.
```

Comment:
```suggestion
It is also possible to simply freeze the software specification. Then, no changes can be made. This would, however, also mean that the identified problems aren't fixed. In the end, this could result in the system not behaving as the user desired. It can also lead to technical debt, as the development team has to somehow navigate around the identified problems.
```